src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsEvoSuiteConfigurable.kt

```diff
@@ -48,8 +47,7 @@ class TestGenieSettingsConfigurable : Configurable {
      */
```

Comment:
Could you please also add @return in the KDoc here as well?

Reply:
done